e2e/transaction_test.go

```diff
@@ -359,13 +359,13 @@ func addStressTestTxns(
 func Test_TransactionDevLoop(t *testing.T) {
 	senderKey, sender := tests.GenerateKeyAndAddr(t)
 	defaultBalance := framework.EthToWei(100)
-	devInterval := 5 // s
+	//devInterval := 1 // s
```

Comment:
Is this leftover?

Reply:
[Removed](https://github.com/0xPolygon/polygon-edge/pull/432/commits/922b56626107993fb1f2a71771297b1758a3e238)